game.html

```diff
@@ -13,17 +14,17 @@
     }
   }
   //guess and answer match
-  if (guess > 95) {
-    message = "You matched! First place for sure!";
+  if ((guess >= 60) && (guess <= 81)){
+    message = "You matched the predicted number of " + answer + ". So.. we're going to have another losing season";
   }
-  else if (60 <= guess >= 81){
-    message = "So.. we're going to have another losing season";
+  else if (guess > 95) {
+    message = "You matched the predicted number of " + answer + ". First place for sure!";
   }
   else if (guess < 60){
-    message = "You matched! so we'll be in last this year...";
+    message = "You matched the predicted number of " + answer + ". So we'll be in last this year...";
   }
   else {
-    message = "Sweet! you matched this super accurate predictor!"
+    message = "You matched the predicted number of " + answer +  ". Sweet! you matched this super accurate predictor!"
   }
    alert(message);
 
```

Comment:
If blocks should be formatted like this.

If (compare) {

} else if {

} else {

}